.github/workflows/CI.yml

```diff
@@ -48,7 +48,7 @@ jobs:
       - name: Install Linux / macOS Conda Packages
         if: contains(matrix.os, 'ubuntu') || contains(matrix.os, 'macos')
         shell: bash -l {0}
-        run: conda install bison=3.4
+        run: conda install bison=3.4 sympy
```

Comment:
In here, let's not install sympy and only run all CPython (and LPython) tests that do not use sympy.

That way a regular LPython development does not depend on sympy in any way, you don't need to care, unless you use those features.

The sympy support will still get tested at the CI, so it won't get broken. But when I run tests locally, we don't depend on sympy.